app/assets/stylesheets/application.scss

```diff
@@ -44,3 +44,18 @@ $govuk-typography-use-rem: false;
 .release__view-diff {
   float: right;
 }
+
+.release__non-production-banner {
+  @include govuk-font(16);
+  margin: govuk-spacing(3);
+  margin-top: 0;
+  padding: govuk-spacing(3);
+  border: 1px solid #B01117;
```

Comment:
Looking good - I missed this on the first pass, please change to
border: 1px solid $govuk-border-colour

Reply:
Thanks!